lib/python/vdm/server/HTTPListener.py

```diff
@@ -1630,6 +1668,101 @@ def delete(username, database_id):
+        # Now start each server
+        __host_name__ = socket.gethostname()
+        __host_or_ip__ = socket.gethostbyname(__host_name__)
+        failed = False
+        server_status = {}
+        for server_id in members:
+            server = [server for server in SERVERS if server['id'] == server_id]
+            curr = server[0]
+            try:
+                if (curr['hostname'] == __host_or_ip__):
+                    deploymentcontents = get_database_deployment(database_id)
+                    retcode = start_local_server(deploymentcontents)
+                    if (retcode == 0):
+                        server_status[curr['hostname']] = 'Success'
+                    else:
+                        failed = True
+                        server_status[curr['hostname']] = 'Error starting server'
+                else:
+                    url = ('http://%s:8000/api/1.0/databases/%u/servers/%u/start') % \
+		                      (curr['hostname'], database_id, server_id)
+                    response = requests.put(url)
+                    if (response.status_code != requests.codes.ok):
+                        failed = True
+                    server_status[curr['hostname']] = json.loads(response.text)['statusstring']
+            except Exception, err:
+                failed = True
+                server_status[curr['hostname']] = str(err)
+
+        if failed:
+            return make_response(jsonify({'statusstring':
+                                          'There were errors starting servers: ' + str(server_status)}),
+                                     500)
+        else:
+            return make_response(jsonify({'statusstring':
+                                          'Start request sent successfully to servers: ' + str(server_status)}),
+                                     200)
+
+class StartServerAPI(MethodView):
+    """Class to handle request to start a server."""
+
+    @staticmethod
+    def put(database_id, server_id):
+        """
+        Starts VoltDB database server on the specified server
+        Args:
+            database_id (int): The id of the database that should be started
+            server_id (int): The id of the server node that is to be started
+        Returns:
+            Status string indicating if the server node was started successfully
+        """
+
+        # TODO: Fix this later. Assume  this is local server for now
+        deploymentcontents = get_database_deployment(database_id)
+        try:
+            retcode = start_local_server(deploymentcontents)
+            if (retcode == 0):
+                return make_response(jsonify({'statusstring': 'Success'}),
+                                             200)
+            else:
+                return make_response(jsonify({'statusstring': 'Error starting server'}),
+                                             500)
+        except Exception, err:
+            return make_response(jsonify({'statusstring': str(err)}),
```

Comment:
There is another module called traceback that we can use to get the stack trace. You think we should pass back the full stack trace?